lib/Runtime/Debug/TTSnapObjects.cpp

```diff
@@ -2163,7 +2163,7 @@ namespace TTD
 
             Js::CallInfo callInfo(static_cast<Js::CallFlags>(generatorInfo->arguments_callInfo_flags), generatorInfo->arguments_callInfo_count, false /*unusedBool*/);
 
-            Js::Arguments arguments(callInfo, (Js::Var*)argVals);
+            Js::Arguments arguments(callInfo, unsafe_write_barrier_cast<Js::Var*>(argVals));
```

Comment:
This matches the pattern from the recent change to lib/Runtime/Library/BoundFunction.cpp